index.js

```diff
@@ -121,7 +121,7 @@ Client.prototype.sync = function (callback) {
     self.memo('sync', memo)
 
     self._log('sync', { reconcileStamp: self.state.reconcileStamp })
-    return self.setTimeUntilReconcile(self.state.reconcileStamp, callback)
+    self.setTimeUntilReconcile(self.state.reconcileStamp, () => {})
```

Comment:
@evq why shouldn't we return here? It's the same case as is on line 129. We set new timestamp and return

Reply:
@NejcZdovc I don't think it's quite the same case.

reconcileStamp should only be null or NaN on first call to sync. before this code path was added, the first sync was responsible for generating the wallet keypair. now that doesn't happen on the first call to sync because of this return. I think the first call to sync should preserve that original behavior (which the integration tests depend on).

i can't see a reason we must return here, wdyt?

Reply:
hmm, make sense.

One think that maybe we can do (so that we don't pass empty function in is):
- add check in `setTimeUntilReconcile` for callback and remove empty function
or
- return here with sync as a callback

wdyt?